docs-js/orchestration/chat-completion.mdx

```diff
@@ -55,13 +55,17 @@ Use the following convenience methods for handling chat completion response:
 - `getContent()` parses the response and returns the model's output as a string.
 - `getFinishReason()` retrieves the `finish_reason` explaining why chat completion request stopped.
 - `getTokenUsage()` provides token usage details, including `total_tokens`, `prompt_tokens`, and `completion_tokens`.
+- `getAllMessages()` parses the response and returns a list of all messages.
+- `getAssistantMessage()` parses the response and returns the assistant message.
 
 ```ts
 const response = await orchestrationClient.chatCompletion();
 
 console.log(response.getContent());
 console.log(response.getFinishReason());
 console.log(JSON.stringify(response.getTokenUsage()));
+console.log(response.getAllMessages());
+console.log(response.getAssistantMessage());
```

Comment:
As they have already been listed above, I see it less meaningful to list them here with console log again. Plus, there will be some more as suggested in the other PR.